plugins/solidigm/solidigm-smart.c

```diff
@@ -0,0 +1,260 @@
+#include <fcntl.h>
```

Comment:
Please add a SPDX-License-Identifier: GPL-2.0-or-later and copyright statement here too.